JPF/bad_buildings.sql

```diff
@@ -348,9 +391,35 @@ drop table if exists bad_buildings;
 
 create table bad_buildings
 select 
-	cc.*, 
+	cc.hbaddress,
+	cc.bbl,
+	cc.buildingid,
+	cc.block,
+	cc.lot,
+	cc.bin,
+	cc.zip,
+	cc.all_hpd_complaints,
+	cc.hs_hpd_complaints, 
 	heat_311_hpd_count,
 	all_311_hpd_count,
+	dob_violation_cnt,
+	permit_cnt,
+	hs_permit_cnt,
+	if (pnyc.unitsres != 0, (cc.all_hpd_complaints / pnyc.unitsres) ,0) all_hpd_complaints_per_resunits,
+	if (pnyc.unitsres != 0, (cc.hs_hpd_complaints / pnyc.unitsres) ,0) hs_hpd_complaints_per_resunits,
+	if (pnyc.unitsres != 0, (heat_311_hpd_count / pnyc.unitsres) ,0) heat_311_hpd_count_per_resunits,
+	if (pnyc.unitsres != 0, (all_311_hpd_count / pnyc.unitsres) ,0) all_311_hpd_count_per_resunits,
+	if (pnyc.unitsres != 0, (dob_violation_cnt / pnyc.unitsres) ,0) dob_violation_cnt_per_resunits,
+	if (pnyc.unitsres != 0, (hs_permit_cnt / pnyc.unitsres) ,0) hs_permit_cnt_per_resunits,
+	if (pnyc.unitsres != 0, (tbcs.unitsstab2007 / pnyc.unitsres) ,0) rentstab2007_per_resunits,
+	if (pnyc.unitsres != 0, (tbcs.unitsstab2015 / pnyc.unitsres) ,0) rentstab2015_per_resunits,
+	if (pnyc.unitsres != 0, (tbcs.diff / pnyc.unitsres) ,0)  diff_per_resunits, 
+	if (pnyc.unitsres != 0, (cvA.classA_cnt / pnyc.unitsres) ,0)  classA_cnt_per_resunits, 
+	if (pnyc.unitsres != 0, (cvB.classB_cnt / pnyc.unitsres) ,0)  classB_cnt_per_resunits, 
+	if (pnyc.unitsres != 0, (cvC.classC_cnt / pnyc.unitsres) ,0)  classC_cnt_per_resunits, 
+	if (pnyc.unitsres != 0, (cvI.classI_cnt / pnyc.unitsres) ,0)  classI_cnt_per_resunits, 		
```

Comment:
Just gonna note this here, as it seems like the best place - we should be careful about including the violations as features in the model if we use the worst landlord list and AEP as the objective, as the direct dependence between the two quantities might lead to a model that simply reproduces the worst landlord list.